### PR TITLE
implement compact switch

### PR DESCRIPTION
**Pull request by petrjasek**
splitting the screen on top (to have 2 subnavs)

### CODE REVIEW

File: client/app/scripts/superdesk-production/views/production.html [hunk 1]
```diff
@@ -1,50 +1,53 @@
-<div class="production-dashboard">
-    <div class ="tabpane left-tabpane open-tabpane widget-aggregate">
-        <div class="tabcontent">
-            <div class="widget-content" style="top:50px;">
+<div class="production-dashboard" ng-init="view = {compact: false}">
+    <div class="list-pane" ng-class="{compact: view.compact}">
+        <div class="subnav">
+            <button class="btn pull-right" ng-class="{active: view.compact}" ng-click="view.compact = !view.compact">compact</button>
```
Comment (plamut) [inline client/app/scripts/superdesk-production/views/production.html]: Does this need to be translated?

Comment (petrjasek) [inline client/app/scripts/superdesk-production/views/production.html]: no there will be an icon, it's just work in progress

Comment (plamut) [inline client/app/scripts/superdesk-production/views/production.html]: Oh, OK then. Adding the label WIP.

**Comment by petrjasek** [comment]
@amagdas @sjunaid what do you think? it adds this extended mode to production view

File: client/app/scripts/superdesk-production/production.js [hunk 2]
```diff
@@ -3,11 +3,58 @@
 
     ProductionController.$inject = ['$scope', 'production', 'superdesk', 'authoring', '$location', 'referrer', '$timeout'];
     function ProductionController($scope, production, superdesk, authoring, $location, referrer, $timeout) {
+        var MAX_VIEW = {max: true},
+            MIN_VIEW = {min: true},
+            ITEM_VIEW = {medium: true},
+            COMPACT_VIEW = {compact: true};
```
Comment (plamut) [inline client/app/scripts/superdesk-production/production.js]: `Object.freeze()` can be used here, too.

**Comment by amagdas** [comment]
:+1:

**Comment by plamut** [comment]
Wait, there were not two thumb-ups yet?

**Comment by amagdas** [comment]
This was against a topic branch, we wanted to take a look at it.
We will always wait for 2 thumbs ups before merging to master.

**Comment by plamut** [comment]
Ah, sorry, completely missed that.

**Pull request merged**